modules/components/core/org.wso2.carbon.pc.core/src/main/java/org/wso2/carbon/pc/core/ProcessStore.java

```diff
@@ -143,6 +151,24 @@ public String createProcess(String processName, String processVersion, String pr
                 appendText(doc, propertiesElement, "bpmnpath", mns, "NA");
                 appendText(doc, propertiesElement, "bpmnid", mns, "NA");
 
+                if(successor.length() != 0){
+                    for(int i = 0 ; i < successor.length() ; i++){
+                        Element successorElement = append(doc, rootElement, "successor", mns);
+                        appendText(doc, successorElement, "name", mns, successor.getJSONObject(i).getString("name"));
+                        appendText(doc, successorElement, "path", mns, successor.getJSONObject(i).getString("path"));
+                        appendText(doc, successorElement, "id", mns, successor.getJSONObject(i).getString("id"));
+                    }
+                }
+
+                if(predecessor.length() != 0){
```

Comment:
Already fixed in https://github.com/wso2/product-pc/pull/36 (See under git comment called "Reformat the code")